justfile

```diff
@@ -298,11 +298,13 @@ build-jupyterlite:
 
     git checkout pyproject.toml ibis/__init__.py uv.lock
 
+    jq '{"PipliteAddon": {"piplite_urls": [$ibis]}}' -nM \
+        --arg ibis "dist/ibis_framework-${ibis_dev_version}-py3-none-any.whl" \
+        > jupyter_lite_config.json
+
     jupyter lite build \
         --debug \
         --no-libarchive \
-        --piplite-wheels "dist/ibis_framework-${ibis_dev_version}-py3-none-any.whl" \
```

Comment:
These flags now throw an error, and the duckdb wheel is no longer required as it ships with the latest pyodide distribution.